.github/workflows/cli-deploy.yml

```
@@ -0,0 +1,39 @@
+        type: string
+jobs:
+  deploy:
+    name: CLI Deploy
+    runs-on: ubuntu-latest
+    environment:
+      name: DEPLOYMENT
+    env:
+      SUBQL_ACCESS_TOKEN: ${{ secrets.SUBQL_ACCESS_TOKEN }}
+      ENDPOINT: ${{ secrets.ENDPOINT }}
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)